src/Symfony/Component/Mime/Part/AbstractMultipartPart.php

```diff
@@ -91,7 +91,7 @@ public function asDebugString(): string
     private function getBoundary(): string
     {
         if (null === $this->boundary) {
-            $this->boundary = '_=_symfony_'.time().'_'.bin2hex(random_bytes(16)).'_=_';
+            $this->boundary = '_symfony_'.time().'_'.bin2hex(random_bytes(16)).'_';
```

Comment:
I'd suggest generating the boundary using:
`$this->boundary = strtr(base64_encode(random_bytes(16)), '+/', '-_'));`
This would prevent leaking "symfony" (someone mentioned the idea elsewhere)
and that would prevent using special characters that must be quoted (as using them reduces interoperability with some servers apparently).

Reply:
For reference purposes, I looked how others generate this boundary value:

* Rust for example uses 6 random alphanumeric ASCII characters: https://docs.rs/multipart-rfc7578/0.7.0/src/multipart_rfc7578/boundary_generator.rs.html#44-49
* Apache uses the object hashcode + some timestamp (https://github.com/apache/camel/blob/abe804645029df960b7299e05546597196aea7a1/components/camel-as2/camel-as2-api/src/main/java/org/apache/camel/component/as2/api/util/EntityUtils.java#L63) e.g. `boundary="----=_Part_0_14180567.1447658227051"`

Reply:
Let's use 6 instead of 16 then? 16 is 128b = good for world-wide uniqueness.
6 = 48b = good for single message uniqueness, as Rust hints us.

Reply:
Updated.